api/src/main/java/graphql/nadel/Nadel.java

```diff
@@ -91,7 +93,8 @@ private Nadel(
             ServiceExecutionHooks serviceExecutionHooks,
             WiringFactory overallWiringFactory,
             WiringFactory underlyingWiringFactory,
-            SchemaTransformationHook schemaTransformationHook
+            SchemaTransformationHook schemaTransformationHook,
+            List<Object> nextGenTransforms
```

Comment:
This works… but honestly we should just feed it directly into the engine e.g.

```java
Nadel.newNadel()
  .engineFactory((nadel) -> {
    return new NextgenEngine(nadel, transforms)
  })
  .build()
```

We don't want to be passing around a `List<Object>`.